video/out/wayland_common.h

```diff
@@ -85,6 +83,12 @@ struct vo_wayland_state {
     int timeout_count;
     int wakeup_pipe[2];
 
+    /* osd surface */
+    struct wl_surface *osd_surface;
+    struct wl_subsurface *osd_subsurface;
+    bool osd_surface_mapped;
+    bool osd_surface_has_contents;
```

Comment:
Small nit but instead of changing this, could you put the two new bools in the in the priv in `vo_dmabuf_wayland` instead since that's going to be the only thing that actually uses this?

Reply:
Good point, done.